### DISCUSSION TITLE
Why Release 6.0.0 Preview 1 has .net462 target?

### DISCUSSION THREAD

**Discussion by alexey-leonovich** [linq2db/linq2db#4698] [Q&A] [answered]
Hello.
Release 6.0.0 Preview 1 [targerts](https://www.nuget.org/packages/linq2db/6.0.0-preview.1#dependencies-body-tab) .net462, .netstandard20, .net60 and .net80.
Why serapate .net462 target is needed? As far as I know .net462 is .netstandard20-[compliant](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0) so any project targeting .net462 will be able to consume .netstandard20 version of linq2db nuget. 
Could you clarify this moment please. Thank you.

**Comment by viceroypenguin** [comment]
From the article you linked regarding netstandard 2.0: 
> The versions listed here represent the rules that NuGet uses to determine whether a given .NET Standard library is applicable. While NuGet considers .NET Framework 4.6.1 as supporting .NET Standard 1.5 through 2.0, there are several issues with consuming .NET Standard libraries that were built for those versions from .NET Framework 4.6.1 projects. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.

There are several key api differences between ns20 and netfx462, and so it is better to support both directly; this way the correct apis are used for each consumer.

**Comment by MaceWindu** [comment, marked as answer]
To be able to use NETFX-only/-native functionality:

- WCF and `System.Data.Services.DataService` remote transports (now WCF has .net support, but we didn't ported it)
- `System.Data.Linq` namespace types support
- app.config connection strings section support (also wasn't ported to .net)
- support for NETFX-only database providers, e.g. `IBM.Data.DB2`, `Sap.Data.Hana.v4.5`
- built-in `System.Data.ODBC`, `System.Data.OleDb` and `System.Data.SqlClient` providers
- workarounds for NETFX-specific issues like https://github.com/dotnet/runtime/issues/6303

**Reply by alexey-leonovich** [reply]
Thanks for the detailed answer.
